tsup.config.ts

```diff
@@ -17,7 +17,7 @@ export default defineConfig((options) => [
     sourcemap: false,
     clean: true,
     platform: 'node',
-    target: 'node16',
+    target: 'node18', // Minimum supported (LTS) version
```

Comment:
I'm not sure we shouldn't compile down to the node versions supported by the versions of SB we support?

Reply:
Fair enough. The difference is 2615 bytes.